05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,42 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+.wrapper {
+    width:1000px;
+    margin:0 auto;
 }
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
-}
-
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.box1 {
+    width:500px;
+    float:left;
+    margin-left: 15px;
 }
 
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+.img {
+    width:500px;
 }
 
```

Comment:
Hey Yvonne, I'm still missing the in-class project for lesson 4:  http://yride.github.io/intro-workbook/04LessonFour/in-class-project/  and the in-class project for the spell-checking the intel page:  http://yride.github.io/intro-workbook/06LessonSix/in-class-project/intel.html  please get these finished and hopefully a start on the homework for today :) let me know if you need any help!